packages/langium-vscode/package.json

```diff
@@ -64,8 +64,7 @@
   },
   "dependencies": {
     "langium": "0.4.0",
-    "vscode-languageclient": "^8.0.1",
-    "vscode-languageserver": "^8.0.1",
+    "vscode-languageserver": "^8.0.2",
```

Comment:
Why did you remove the client lib here? We use it in extension.ts

Reply:
@msujew what about this? Seems to work anyway because of hoisted dependencies, but it seems unclean.

Reply:
Ups, forgot about this. I'll create another PR.